mlrun/model_monitoring/stream_processing.py

```diff
@@ -137,6 +137,12 @@ def _initialize_v3io_configurations(
         self.tsdb_batching_max_events = tsdb_batching_max_events
         self.tsdb_batching_timeout_secs = tsdb_batching_timeout_secs
 
+        self.tsdb_configurations = {
+            "access_key": self.v3io_access_key,
```

Comment:
Not specifically regarding the addition here, but this is no way to treat an access key...